### DISCUSSION TITLE
GOtoGO update

### DISCUSSION THREAD

**Discussion by Romek1234** [GrandOrgue/grandorgue#729] [Help] [answered]
How do I update to the latest version of Grandorgue Live?

**Comment by rousseldenis** [comment]
Hi @Romek1234, I see the last Live version is from 2016.

We have not yet thought about Live version since the GitHub move.

@larspalo @oleg68 do we need to plan such resurrection?

**Reply by oleg68** [reply]
I think we should plan it as well as to support GO repositories for different linux distributions like Martin used to do with OBS.

**Comment by larspalo** [comment, marked as answer]
GOtoGO was a live distribution (based on openSUSE, if I remember correctly) specifically tailored to be efficient at running GO. It was actually quite pleasant to use and fairly easy for someone not accustomed to Linux. Martin created that with everything pre installed for the user - you just added additional samplesets that you wanted and was ready to go... The only drawback for us could be needing to host a fairly big .iso file and keep update handling - especially of GO - relatively easy. Personally, I think it could be interesting to have a custom lubuntu based image - or if one goes even further: a DSL based one! Just dreaming, but at one point in time I think I actually had GO running on DSL.

**Reply by rousseldenis** [reply]
Yes.

The day we'll integrate automatic updates will easy such things

**Comment by Romek1234** [comment]
Is it possible after installing GO Live on the disk, update the GO itself, e.g. with a zypper ... or something like that?
I don't know Linux well, but I really like GO Live. It would be good to use it in the current version.

**Reply by oleg68** [reply]
You may try:
1. Download appropriate rpms
2. zypper update .…list of rpms...

But it may fail if there is no required dependencies in Go live.

**Reply by Romek1234** [reply]
Oh thanks. I didn't know it's that simple.

**Reply by Romek1234** [reply]
I installed GO Live on hdd from the GOtoGO.x86_64-2016.05.15.iso image downloaded as usual from sourceforge: https://sourceforge.net/projects/ourorgan/files/GOLive 

Then I downloaded the rpm package grandorgue-0.3.1.2334-lp153.155.3.x86_64 from GitHab: https://github.com/GrandOrgue/grandorgue/actions/runs/1273503636

Attempting to install zypper install ... has failed. 
Installation with Yast2 also failed. No proper library available. libm.so.6 (GLIBC_2.27) (64bit)
YaST2 message:
YaST2 conflicts list - generated 2021-09-27 12:05:43 ####
no package provides libm.so.6 (GLIBC_2.27) (64bit) needed by  #grandorgue-0.3.1.2334-lp153.155.3.x86_64
Unfortunately I was unable to find and download this library.

AND Attempting to install the new grandorgue-3.2.0-0.15.x86_64.rpm package
with Githab failed. This time the YaST2 message:
### YaST2 conflicts list - generated 2021-09-27 13:25:58 ####
no package provides ibmvec.so.1 () (64bit) needed by grandorgue-3.2.0-0.15.x86_64
#### YaST2 conflicts list END ###

I found it on software.opensuse.org and downloaded the rpm package grandorgue-0.3.1.2334-159.1.x86_64 from https://software.opensuse.org/download/package?package=grandorgue&project=home%3Ae9925248%3Abranches%3AopenSUSE%3ALeap%3A42.1%3AUpdate
You have to click "Download binary packages directly" and download https://download.opensuse.org/repositories/home:/e9925248:/branches:/openSUSE:/Leap:/42.1:/Update/standard/x86_64/grandorgue-0.3 .1.2334-159.1.x86_64.rpm

YaST2 installed this package without any problems and I can enjoy GrandOrgue version 0.3.1.2334

Thank you all and best regards!
